api/tests/Unit/v2/Registration/Uploader/OrgDeputyshipUploaderTest.php

```diff
@@ -271,6 +271,53 @@ public function uploadClientAndNamedDeputyAreNotAssociatedWhenClientHasSwitchedO
         self::assertEquals($originalNamedDeputy->getDeputyUid(), $updatedClient->getNamedDeputy()->getDeputyUid());
     }
 
+    /** @test  */
+    public function uploadClientAndNamedDeputyAreAssociatedWhenClientHasSwitchedOrgsAndNamedDeputyHasNotChanged()
```

Comment:
So if I'm understanding it rightly you've added two code blocks for "not new court order" and "not switched org", but your test here is only covering the has switched org case. Is there an existing test for the other scenario combinations?